src/vuestic-theme/vuestic-components/vuestic-breadcrumbs/VuesticBreadcrumbs.vue

```diff
@@ -3,15 +3,15 @@
     <div class="col-md-12">
       <vuestic-widget class="no-padding no-v-padding">
         <div class='vuestic-breadcrumbs'>
-          <router-link :to="'/'" class="crumb">{{'breadcrumbs.home' | translate}}</router-link>
+          <router-link :to="'/'" class="crumb">{{$t('breadcrumbs.home')}}</router-link>
```

Comment:
- home path must be also configurable via configuration object (might be multiple routes, or not always '/')

- create app-breadcrumbs component just like app-navbar/app-sidebar

- move .row and .col to app-breadcrumbs component

- leave widget as a root component for vuestic-breadcrumbs, with .vuestic-breadcrumbs class attached to the widget component

- move breadcrumbs.js to app-breadcrumbs folder instead of data folder